### ISSUE TITLE
Floating simulator panel in notebook

### ISSUE THREAD

**Issue by psychemedia** [innovationOUtside/nbev3devsim#41]
It would be useful if we could detach the simulator from a notebook output cell and display it either in something like the pager at the bottom of the notebook in an expandable/collapsible area, or have it in a floating panel, eg as discussed in https://github.com/innovationOUtside/nbev3devsim/issues/30#issuecomment-617585446

See also [this example](https://github.com/jupyter-widgets/ipywidgets/issues/2520) of a torn off output widget floating at the top of a notebook which uses the following setup in part:

```python
display(Javascript("""$('div.job_widget')
        .detach()
        .appendTo($('#header'))
        .css({
            'z-index': 999,
             'position': 'fixed',
            'box-shadow': '5px 5px 5px -3px black',
            'opacity': 0.95,
            'float': 'left,'
        })
        """))
```

See also the [`ipyturtle` widget](https://github.com/gkvoelkl/ipython-turtle-widget/blob/master/js/lib/example.js), which has a draggable panel, as does the [notebook variable inspector extension](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/varInspector).

Related issue: https://github.com/AaronWatters/jp_proxy_widget/issues/8

**Comment by psychemedia** [comment]
As per https://github.com/AaronWatters/jp_proxy_widget/issues/8, we can display and detach the simulator as a floating panel using:

```python
roboSim = eds.Ev3DevWidget()
display(roboSim)
roboSim.element.dialog()
```
